Installers/default.build

```diff
@@ -173,6 +185,16 @@ fi
                 <include name="*.*"/>
          </fileset>
       </copy>
+      <copy todir="Linux/tmp_run/MonoGameSDK/Assemblies/DesktopGL/x86">
```

Comment:
The line above (`<copy todir="Linux/tmp_run/MonoGameSDK/Assemblies/DesktopGL">`) already copies the native files for Linux installer.